ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -1,3 +1,4 @@
+# For Review
```

Comment:
we actually shouldn't need to test for `as_array` instance method, this is a pointless ivy.Array instance method, as it will always simply return itself. This will be removed, and so this can be hard-coded as `False` in the test.

Reply:
This makes sense